### ISSUE TITLE
High Training PSNR but Low Validation PSNR and Color Errors in Rendered Images

### ISSUE THREAD

**Issue by SensenGao** [OpenRobotLab/AnySplat#60]
<img width="1337" height="712" alt="Image" src="https://github.com/user-attachments/assets/16d1f321-f20b-4800-a3e6-04624e546668" />

Although the PSNR during training is as high as 29, I don't know why the rendered images always have incorrect colors, and the PSNR on the validation set is very low, only 13.

**Comment by GitHubTheSun** [comment]
I also observed this phenomenon. 
By reading the training code of anysplat, I found that the initial image loading sets the range to [-1, 1]. However, in _train_step_, the image tensor normalized to the range [0, 1] is passed to _self.model_, while in _validation_step_, the image remaining in the range [-1, 1] is passed to _self.model_. As a result, this causes incorrect image colors during rendering and leads to a lower PSNR.

**Comment by GitHubTheSun** [comment]
Well, your training results look much better than mine. I initially suspect this might be because I used too small a dataset. So, I’d like to ask about the details of your training: what kind of dataset did you use, and how many scenarios were included? Thank you!

**Comment by SensenGao** [comment]
I used re10k, not DL3DV. Thank you for your answers to my questions.

**Comment by SensenGao** [comment]
The performance of DL3DV doesn’t seem to be very good in general, and I can only reach around 19–20 in terms of PSNR.

**Comment by GitHubTheSun** [comment]
OK. I will try using a new dataset in order to achieve better training results. Thank you~

**Comment by jianglh-WHU** [comment]
Thanks for your observation, I will fix that bug in validation_step!

**Comment by terencepp** [comment]
> Thanks for your observation, I will fix that bug in validation_step!

Hello, could you please tell me have you fix that?

**Comment by Giperx** [comment]
> > Thanks for your observation, I will fix that bug in validation_step!
> 
> Hello, could you please tell me have you fix that?

You can just modify this line
https://github.com/InternRobotics/AnySplat/blob/57937e4e29a30ed65e86d8813edd4a4797b0696c/src/model/model_wrapper.py#L476
to
```python
encoder_output, output = self.model((batch["context"]["image"] + 1) / 2, self.global_step, visualization_dump=visualization_dump)
```

**Comment by Alex417ii** [comment]
I also use the Dl3DV-10k-samples dataset, but during training, I encountered unstable or even increasing loss. Could you please share your parameter configuration and any other modifications you made? Thank you very much!

> I used re10k, not DL3DV. Thank you for your answers to my questions.
